.github/workflows/build-verification.yml

```diff
@@ -25,3 +25,69 @@ jobs:
         run: ./gradlew build -i
         env:
           GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_SOLUTIONS_ACCESS_TOKEN }}
+      - name: Download and extract build validation Scripts
```

Comment:
Are these additions intentional?

Reply:
Indeed. I feel like we're missing simple tests like this for our PR branches. Though the name of this step is incorrect and has been fixed.

Let me know if you think I should remove these tests.

Reply:
I don't have any objection to them, so long as they don't take a crazy amount of time to run (like a an hour or two).

Reply:
The run for this PR took only 2 minutes. 😄 

https://github.com/gradle/gradle-enterprise-build-validation-scripts/actions/runs/3680308034/jobs/6225766497